stable/aws-events-exporter/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for aws-events-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicas: 1
+
+image:
+  repository: thomasnyambati/aws-events-exporter
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: 1.0.0
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+# serviceAccountName -- service account to be used by the containers
+serviceAccountName: ""
+podAnnotations: {}
```

Comment:
If you install this chart as is, prom will never find the `/metrics` endpoint as the default annotations are not present either on the service or on the pod.

I think it's better to include the required pod annotations by default and make the service optional, like here: https://github.com/deliveryhero/helm-charts/blob/master/stable/prometheus-soti-mobicontrol-exporter/values.yaml#L55-L66